MarkdownApp/Shared/ViewModels/LoginViewModel.cs

```diff
@@ -0,0 +1,44 @@
+﻿using MarkdownApp.Shared.Models;
+using MarkdownApp.Shared.ViewModels.Interfaces;
+using System.ComponentModel.DataAnnotations;
+using System.Net.Http.Json;
+using System.Text.Json.Serialization;
+
+#nullable disable
+
+
+namespace MarkdownApp.Shared.ViewModels
+{
+    public class LoginViewModel : ILoginViewModel
```

Comment:
you model is a model and service and api client in same time
it's a not a good practice
you have to save single responsibility of you classes

Reply:
no, the idea was to create a repository or service rather than "viewModels" to call the controller, so the viewmodel have no place in here.
I tried to put in the shared project : model, authentification service (it needs seperation from the model)
Server for Apis
Client for UI.